---> QA-60(First Code Push) <---

karnanqa opened the pull request:
CreateJOb.java: Include JobCostPO method
Customer.java: Include Add_Customer method
Karna.java: Include PO method
searchengine.java: Included searchbox_jobCostSuppliersearch method

Ranjit kindly review and revert me if any corrections.

Commit:
---> QA-60(First Code Push) <---
CreateJOb.java: Include JobCostPO method
Customer.java: Include Add_Customer method
Karna.java: Include PO method
searchengine.java: Included searchbox_jobCostSuppliersearch method

Ranjit kindly review and revert me if any corrections.

Ranjit-RR reviewed (commented):
Hi karna..,
 select supplier locator needs to be changed can you have a look at the "search engine" page object model for your reference in the "Pages" package and so this needs to update before merging in the Smoke test and job module.

Commit:
---> QA-60(Second Code Push) <---
CreateJOb.java: Include JobCostPO method
Karna.java: Include PO method
searchengine.java: Included searchbox_jobCostSuppliersearch method

Ranjit kindly review and revert me if any corrections.

Create Customer - Create Job - Create PO

Commit:
Just added one thread.sleep for the  wait to find the element
